src/Data/Repositories/Repository.cs

```diff
@@ -18,115 +15,38 @@ protected Repository(ApplicationContext context)
             Set = context.Set<TEntity>();
```

Comment:
rename Set to dbSet, as for me, the name of Set is not entirely intuitive

Reply:
Done